Users/coafile.rst

```diff
@@ -39,6 +39,10 @@ wide coafile which has the lowest priority. The ``default_coafile`` must
 lie in the coala installation directory and is valid for everyone using
 this coala installation.
 
+It can be used to define the type of files you usually don't want to lint
+like minified files (e.g. ``*.min.js``) and backup files (e.g. ``*.orig``)
+such as the file present in ``.coafile`` in the main coala repository.
```

Comment:
"such as the file present in ``.coafile``" doesn't make sense IMO. You wished to convey that how .coafile contains a list of files that are ignored and are not linted, you can specify the same in default_coafile, right? It's not very clear right now.

Reply:
okk so sud i remove that line?

Reply:
No, just rephrase it.